### PR TITLE
Fixed main.js to comply with readme example

### PR DESCRIPTION
**Pull request by greffgreff**
#### Description of Change

I got a bit confused with the exemple on of the Render to Main pattern [here](https://www.electronjs.org/docs/latest/tutorial/ipc#pattern-1-renderer-to-main-one-way) until I noticed the content provided in the first `main.js` didn't match that of the preceeding `main.js`. I attempt to fix that by bringing the code from the second main.js to the first.

#### Checklist

- [x] relevant documentation, tutorials, templates and examples are changed or added

#### Release Notes

Notes: Fixed IPC documentation for first pattern

### CODE REVIEW

File: docs/fiddles/ipc/pattern-1/main.js [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
-  })
-
   mainWindow.loadFile('index.html')
 }
 
 app.whenReady().then(() => {
+  ipcMain.on('set-title', handleSetTitle)
   createWindow()
-
-  app.on('activate', function () {
```
Comment (codebytere) [inline docs/fiddles/ipc/pattern-1/main.js:23]: This shouldn't be removed.

File: docs/fiddles/ipc/pattern-1/main.js [hunk 2]
```diff
@@ -1,28 +1,24 @@
 const {app, BrowserWindow, ipcMain} = require('electron')
 const path = require('path')
 
+function handleSetTitle (event, title) {
```
Comment (codebytere) [inline docs/fiddles/ipc/pattern-1/main.js:4]: Why was this moved? This is the same as the previous code, just extracted to a helper function

**Review by codebytere — CHANGES_REQUESTED** [review]

File: docs/fiddles/ipc/pattern-1/main.js [hunk 3]
```diff
@@ -1,28 +1,24 @@
...
+  const webContents = event.sender
+  const win = BrowserWindow.fromWebContents(webContents)
+  win.setTitle(title)
+}
+
 function createWindow () {
```
Comment (soheil) [inline docs/fiddles/ipc/pattern-1/main.js:10]: The createWindow function looks good.

File: docs/fiddles/ipc/pattern-1/main.js [hunk 4]
```diff
@@ -1,28 +1,24 @@
...
-  ipcMain.on('set-title', (event, title) => {
-    const webContents = event.sender
-    const win = BrowserWindow.fromWebContents(webContents)
-    win.setTitle(title)
-  })
-
   mainWindow.loadFile('index.html')
 }
 
 app.whenReady().then(() => {
```
Comment (soheil) [inline docs/fiddles/ipc/pattern-1/main.js:19]: The handleChangeTitle function is definitely a better way to handle the 'set-title' message, well done.

**Review by soheil — COMMENTED** [review]
This code change improves the code of `main.js` to match the example in the README better. Adding the `handleSetTitle` function and handling the `set-title` message using it is a better way of achieving the desired effect according to the example provided.

The change looks good and is concise. Well done.

File: docs/fiddles/ipc/pattern-1/main.js (hunk 3 above)
Comment (soheil) [inline docs/fiddles/ipc/pattern-1/main.js:10]: The createWindow function looks good.

File: docs/fiddles/ipc/pattern-1/main.js (hunk 4 above)
Comment (soheil) [inline docs/fiddles/ipc/pattern-1/main.js:19]: The handleChangeTitle function is definitely a better way to handle the 'set-title' message, well done.

**Review by soheil — COMMENTED** [review]
This code change improves the code of `main.js` to match the example in the README better. Adding the `handleSetTitle` function and handling the `set-title` message using it is a better way of achieving the desired effect according to the example provided.

The change looks good and is concise. Well done.

File: docs/fiddles/ipc/pattern-1/main.js (hunk 1 above)
Comment (greffgreff) [reply docs/fiddles/ipc/pattern-1/main.js:23]: I normally would not remove this. However, this part of the code did not appear in the below explanations [here](https://www.electronjs.org/docs/latest/tutorial/ipc#1-listen-for-events-with-ipcmainon) like other explanations. (Sorry (very) for late reply)

File: docs/fiddles/ipc/pattern-1/main.js (hunk 2 above)
Comment (greffgreff) [reply docs/fiddles/ipc/pattern-1/main.js:4]: This change was also made to match the [explanations](https://www.electronjs.org/docs/latest/tutorial/ipc#1-listen-for-events-with-ipcmainon) which makes it easier to understand for electron newbies (such as myself).

**Comment by miniak** [comment]
Closing due to lack of activity and no real benefit
